common/app/model/Formats.scala

```diff
@@ -198,9 +198,31 @@ object ContentTypeFormat {
   implicit val genericThriftAtomFormat = GenericThriftAtomFormat
   implicit val recipeThriftAtomFormat = RecipeThriftAtomFormat
   implicit val reviewThriftAtomFormat = ReviewThriftAtomFormat
+  implicit val storyquestionsThriftAtomFormat = StoryquestionsThriftAtomFormat
   implicit val recipeAtomFormat = Json.format[RecipeAtom]
   implicit val reviewAtomFormat = Json.format[ReviewAtom]
-  implicit val atomsFormat = Json.format[Atoms]
+  implicit val storyquestionsAtomFormat = Json.format[StoryQuestionsAtom]
```

Comment:
cc @TBonnin to get your thoughts. This change would mean we don't have to repress fronts just to add a field for an atom.

I am happy with this change, but we have added it to our team's goals to understand and potentially come up with a better architecture for atoms in frontend, as adding it as part of facia seems a bit hacky.